default.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 4</title>
+    <script src="jquery-3.4.1.min.js"></script>
```

Comment:
This is not the name of your jquery file in your repo. As cloned - your script will not run because of this inconsistency.
-2